STATIC_CONTENT_MANIFEST build error (worker bundle fail)

## Description

I used Docker to set up the development environment for this project and followed the README to run `yarn start` (after running `yarn install`), but I kept encountering errors related to `__STATIC_CONTENT_MANIFEST` during startup. This caused the Cloudflare Worker to fail to bundle correctly, and the website could not start.

## Summary of Process

- Used your `Dockerfile` to build the container, entered `/workspace`, and ran `yarn install`.
- Tried `yarn start` as per the README (also tried `npx wrangler dev` and `./build.sh --dev`).
- During the build phase, trunk + worker-build generated wasm and js; the `worker/build/worker` directory contains `index.wasm`, `index_bg.js`, and `shim.js` (the build structure is shown in the attached diagram).
- When running `yarn start` / wrangler build, esbuild tries to resolve the Cloudflare virtual module in `index_bg.js`, resulting in the following error:

```
✘ [ERROR] Could not resolve "__STATIC_CONTENT_MANIFEST"
index_bg.js:1:36:
 1 │ import { default as default1 } from '__STATIC_CONTENT_MANIFEST';
```

<img width="1660" height="2160" alt="Image" src="https://github.com/user-attachments/assets/5470e2bf-e573-48bc-a88f-77b85b70383a" />

## Development Environment (Dockerfile)

Below is the `Dockerfile` I created:

```dockerfile
# Base image: official Rust + Node.js
FROM rust:latest

# Install Node.js and npm
RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
    apt-get install -y nodejs

# Install Trunk, wasm-pack
RUN cargo install trunk && \
cargo install wasm-pack && \
cargo install worker-build

# When I tried to install this package, I found it was deprecated and recommended to use npm instead
# RUN cargo install wrangler

RUN npm install -g yarn wrangler

# Install wasm32-unknown-unknown toolchain
RUN rustup target add wasm32-unknown-unknown && \
    rustup toolchain install nightly

# Set default working directory
WORKDIR /workspace

# Default to bash shell for manual commands
CMD ["bash"]
```

### Questions

1. I'm not sure if `yarn start` is the correct way to start the project (the README suggests this, but I'm not sure if it triggers all necessary build steps).
2. I can start the frontend server and load wasm/js resources using `trunk serve`, but the page only shows `%app%` (see attached image). I'm not sure if this is because the worker wasn't bundled correctly, or if it's a Sycamore hydration/DOM injection issue.
3. Does the version of `wrangler` installed in the `Dockerfile` affect the startup and build of the project? Is there anything I should pay attention to regarding version compatibility? When I tried `cargo install wrangler`, I was notified it was deprecated and recommended to use npm (`npm install -g wrangler`).

<img width="1919" height="930" alt="Image" src="https://github.com/user-attachments/assets/8a5177b8-f2b2-4a44-b7e9-2645732df551" />

## Hardware (user provided)

- OS: macOS Tahoe 26.0 beta
- Device: Mac M4 Max

--- a/README.md
+++ b/README.md
@@ -22,10 +22,8 @@ Required dependencies:
 * Node 16.7+
 * Yarn
 * Latest Rust (including wasm toolchain: `rustup target add wasm32-unknown-unknown`)
-* wrangler: `cargo install wrangler`
 * TrunkRS: `cargo install trunk`
-* worker-build: `cargo install worker-build`
-* wasm-pack: `cargo install wasm-pack`
+* worker-build: `cargo install --git https://github.com/Dav1dde/workers-rs.git --branch pobbin worker-build`
 
 The application can be started with:
 

